pkg/identityserver/store/oauth.go

```diff
@@ -58,6 +58,8 @@ type AuthorizationCode struct {
 	User   *User
 	UserID string `gorm:"type:UUID;index;not null"`
 
+	UserSessionID *string `gorm:"type:UUID;index"`
```

Comment:
This is a `bump/minor` API change

Reply:
That's a not an API right? It's an internal model used by the IS. A migration should be sufficient.

Reply:
Right, I meant storage. It's not clear in the compatibility commitment, I'll clarify that. But we don't do database migrations in patches.